### PR TITLE
Sku can be used to identify a product variant for an update action, close #1023

### LINKED ISSUE #1023: a sku can be used to identify a Product Variant when updating a Product
see  Release Notes - commercetools™ platform - Version Release 23.09.2015 https://dev.commercetools.com/release-notes.html#release-2015-09-23

* [ ] add badge

### PR DESCRIPTION
**Pull request by ccsalazarr-commercetools**
@yanns could you review this please.

### CODE REVIEW

File: commercetools-models/src/main/java/io/sphere/sdk/products/commands/updateactions/RemoveImage.java [hunk 1]
```diff
@@ -12,30 +14,58 @@
...
+
+    public static RemoveImage ofSku(final Image image, final String sku) {
+        return ofSku(image.getUrl(), sku);
+    }
+
+    public static RemoveImage ofVariantId(final String imageUrl, final Integer variantId) {
+        return new RemoveImage(imageUrl, variantId, null);
     }
 
-    public static RemoveImage of(final String imageUrl, final Integer variantId) {
```
Comment (yanns) [inline commercetools-models/src/main/java/io/sphere/sdk/products/commands/updateactions/RemoveImage.java]: Please keep this as well.

File: commercetools-models/src/main/java/io/sphere/sdk/products/commands/updateactions/RemoveVariant.java [hunk 2]
```diff
@@ -4,32 +4,54 @@
...
     public static RemoveVariant of(final ProductVariant variant) {
-        return of(variant.getId());
+        return ofVariantId(variant.getId());
+    }
+
+    public static RemoveVariant ofVariantId(final Integer id) {
+        return new RemoveVariant(id, null);
     }
 
-    public static RemoveVariant of(final Integer id) {
```
Comment (yanns) [inline commercetools-models/src/main/java/io/sphere/sdk/products/commands/updateactions/RemoveVariant.java]: please keep this.

File: commercetools-models/src/main/java/io/sphere/sdk/products/commands/updateactions/SetAttribute.java [hunk 3]
```diff
@@ -62,9 +102,28 @@ public static SetAttribute of(final Integer variantId, final String name, @Nulla
...
+        return ofVariantId(variantId, name, null);
     }
 
+    /**
+     * Action to remove a custom attribute.
+     * @param variantId the variant the attribute value should be unset
+     * @param name the name of the attribute, consult the product type to find the name
+     * @return update action
+     */
+    public static SetAttribute ofVariantIdUnsetAttribute(final Integer variantId, final String name) {
```
Comment (yanns) [inline commercetools-models/src/main/java/io/sphere/sdk/products/commands/updateactions/SetAttribute.java]: Suggestion for another names: `ofUnsetAttributeForVariantId` and `ofUnsetAttributeForSku`?

**Comment by ccsalazarr-commercetools** [comment]
@yanns the changes are made

**Comment by yanns** [comment]
👍

**Pull request merged**